modules/ballerina-test/src/test/java/org/ballerinalang/test/types/map/BMapValueTest.java

```diff
@@ -184,4 +186,47 @@ public void testBMapToString() {
         
         Assert.assertEquals(map.stringValue(), "{\"key1\":1, \"key2\":\"foo\", \"key3\":<bar>hello</bar>}");
     }
+
+    @Test(dependsOnMethods = "testGrammar")
+    public void testMapOrder() {
+//        BValue[] returnVals = BLangFunctions.invokeNew(programFile, "testMapOrder", new BValue[0]);
+        BValue[] returnVals = BRunUtil.invoke(programFile, "testMapOrder", new BValue[0]);
+        BMap m = (BMap) returnVals[0];
```

Comment:
You must not use the raw type, but use the parameterized types, such as BMap<String, String>, Set<String> and all. These are mainly given as IDE warnings, there should be 0 warnings. 

Fix in all the places.